fili-core/src/main/java/com/yahoo/bard/webservice/application/AbstractBinderFactory.java

```diff
@@ -302,6 +303,8 @@ protected void configure() {
 
                 bind(getClock()).to(Clock.class);
 
+                bind(getHttpResponseMaker()).to(HttpResponseMaker.class);
```

Comment:
This needs a test. At least testing that it's been binded like `1 * dc.bind({ it.implementation.contains(HttpResponseMaker.canonicalName) }, _)` in `"test configure bindings"()`